### PR TITLE
PHPStan (Level 1) on controllers/

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | PHPStan (Level 1) on controllers/
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Relative to #16471 
| How to test?      | Github Actions are :green_circle: 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26770)

### CODE REVIEW

File: controllers/admin/AdminImportController.php [hunk 1]
```diff
@@ -4260,108 +4264,108 @@ protected function supplyOrdersDetailsImportOne($info, &$products, &$reset, $for
...
-                    $id_product,
-                    $id_product_attribute,
-                    $current_line + 1
-                );
-            } else {
-                $products[$id_product][$id_product_attribute] = $quantity_expected;
-            }
+        // sets parameters
+        $id_product = (int) $info['id_product'];
+        if (!$info['id_product_attribute']) {
```
Comment (sowbiba) [inline controllers/admin/AdminImportController.php]: why not `!isset($info['id_product_attribute'])` ?

File: controllers/admin/AdminProductsController.php [hunk 2]
```diff
@@ -1440,24 +1438,27 @@ public function ajaxPreProcess()
     public function ajaxProcessUpdateProductImageShopAsso()
     {
         $id_product = Tools::getValue('id_product');
-        if (($id_image = Tools::getValue('id_image')) && ($id_shop = (int) Tools::getValue('id_shop'))) {
+        $id_image = Tools::getValue('id_image');
```
Comment (sowbiba) [inline controllers/admin/AdminProductsController.php]: id_image is casted below. Shouldn't we cast it here ?

**Review by sowbiba — COMMENTED** [review]
some suggestions

**Review by matks — APPROVED** [review]

File: controllers/admin/AdminProductsController.php (hunk 2 above)
Comment (Progi1984) [reply controllers/admin/AdminProductsController.php]: Fixed

File: controllers/admin/AdminImportController.php (hunk 1 above)
Comment (Progi1984) [reply controllers/admin/AdminImportController.php]: Fixed

**Review by sowbiba — APPROVED** [review]

**Comment by sowbiba** [comment]
Thanks @Progi1984

**Pull request merged**